hibernate-core/src/main/java/org/hibernate/metamodel/mapping/internal/SelectableMappingImpl.java

```diff
@@ -286,7 +286,7 @@ public String getSelectionExpression() {
 
 	@Override
 	public String getSelectableName() {
-		return selectablePath == null ? null : selectablePath.getSelectableName();
+		return selectablePath.getSelectableName();
```

Comment:
I'm a little confused here... You explicitly marked `selectablePath` as nullable, but then removed the null handling?

Reply:
The constructor argument is `@Nullable`, but the field is non-null since that is handled in the constructor.

> `this.selectablePath = selectablePath == null ? new SelectablePath( selectionExpression ) : selectablePath;`